src/main/java/uk/gov/hmcts/darts/common/exception/AuditApiError.java

```diff
@@ -17,6 +17,11 @@ public enum AuditApiError implements DartsApiError {
         "101",
         HttpStatus.BAD_REQUEST,
         "When using date filters, both must be provided."
+    ),
+    NO_HEARING_OR_USER_FOUND_WHEN_ADDING_AUDIO_AUDIT(
+        "101",
+        HttpStatus.BAD_REQUEST,
+        "No hearing or user found when auditing add audio."
```

Comment:
should `add audio` either be one word or surrounded by quotes?

Reply:
Ill change it to No hearing or user found when auditing 'add audio' request.